<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Address_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> 주소 : </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Address_textedit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;127.0.0.1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
    <property name="placeholderText">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>이름 : </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Name_textedit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>접속</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Chat_plaintextedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="ChatSend_textedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ChatSend_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>전송</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FileSend_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 전송</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
